data/19.0.0.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<release xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../schema.xsd" version="18.0.0">
+  <changelog href="https://nextcloud.com/changelog/#19-0-0"/>
+  <whatsNew lang="en">
+    <regular>
+      <item>Talk 9 integrates document viewing/editing and a grid view</item>
```

Comment:
Users can't enable talk, or talk will tell it themselves, so this should be moved to admin?

Reply:
Well, if users don't have Talk installed, they can ask the sysadmin... I'm happy to advertise Talk a little this way. You disagree?